app/controllers/variable_drafts_collection_searches_controller.rb

```diff
@@ -0,0 +1,37 @@
+class VariableDraftsCollectionSearchesController < CmrSearchController
+
+  def new
+    @draft = VariableDraft.find(params[:id])
+    add_top_level_breadcrumbs
+
+    unless @draft.collection_concept_id.blank?
+      current_collection_response = cmr_client.get_collections_by_post(
+        # for error, use collection_concept_id as param key
```

Comment:
clarify comment?

Reply:
it was only for testing the error scenarios, it has been removed